cmd/juju/config/config.go

```diff
@@ -165,21 +146,43 @@ func sliceContains(slice []string, val string) bool {
 	return false
 }
 
-// checkSingleAction checks that exactly one action has been specified.
-func (c *ConfigCommandBase) checkSingleAction() error {
-	switch len(c.Actions) {
-	case 0:
-		return errors.New("no action specified")
-	case 1:
+// validateActions checks that the requested combination of actions is valid
+// (i.e. we permit doing these actions simultaneously). The rules are:
+//   - Set & Reset can be done simultaneously, as long as we don't try to
+//     set and reset the same key;
+//   - SetFile can be done simultaneously with Set and/or Reset - in this case,
+//     the Set/Reset arguments will override anything specified in the file;
+//   - GetOne cannot be done along with any other action.
+func (c *ConfigCommandBase) validateActions() error {
```

Comment:
This does more than validate which you'd expect to be read only - it also sets Actions to GetAll.
Can we do this bit before calling validate
```
if len(c.Actions) == 0 {
	// Nothing has been specified - so we get all
	c.Actions = []Action{GetAll}
	return nil
}
```